csr/csr.go

```diff
@@ -49,7 +49,7 @@ type BasicKeyRequest struct {
 
 // NewBasicKeyRequest returns a default BasicKeyRequest.
 func NewBasicKeyRequest() *BasicKeyRequest {
-	return &BasicKeyRequest{"ecdsa", curveP256}
+	return &BasicKeyRequest{"rsa", 2048}
```

Comment:
@kayrus I think a loop that appends if the trimmed string length is non-zero would probably work best.

As for the ECDSA default, I'll add a new documentation file discussing certificate requests in CFSSL. Do you think that would be a good place for them?